microshift_support/microshift-version.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="microshift-version"]
+= MicroShift Version
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-version
+toc::[]
+
+[id="microshift-version-cli"]
+== Checking the MicroShift Version from the command line
+
+From the command line, use `microshift version` to check the version information.
```

Comment:
I realize this hasn't been broken up into its own procedure module just yet, but even single step procedures are itemized:
```suggestion
.Procedure
* Run the following command to check the version information:
+
```

Reply:
The proposed change also removes the text that comes before the procedure. Should that be retained in another way or is it superfluous?

Reply:
It should be retained, but as the step instruction. I tried to reflect this in the sentence proposed for line 12